internal/controller/controllers/bmo_utils.go

```diff
@@ -0,0 +1,135 @@
+package controllers
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/coreos/go-semver/semver"
+	v1 "github.com/openshift/api/config/v1"
+	metal3iov1alpha1 "github.com/openshift/cluster-baremetal-operator/api/v1alpha1"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+type BMOUtils struct {
+	client.Client
+	log            logrus.FieldLogger
+	kubeAPIEnabled bool
+}
+
+func NewBMOUtils(client client.Client, log logrus.FieldLogger, kubeAPIEnabled bool) *BMOUtils {
+	return &BMOUtils{client, log, kubeAPIEnabled}
+}
+
+// ConvergedFlowAvailable checks the baremetal operator version and returns true if it's equal or higher than the minimal version for converged flow
+func (r *BMOUtils) ConvergedFlowAvailable(minimalVersionForConvergedFlow string) bool {
```

Comment:
If we make `minimalVersionForConvergedFlow` a constant this function won't need to receive any parameter.